config/samples/xgboost-dist/README.md

```diff
@@ -248,3 +249,242 @@ Events:
   Normal  ExitedWithCode           38s                xgboostjob-operator  Pod: default.xgboost-dist-iris-test-predict-master-0 exited with code 0
   Normal  XGBoostJobSucceeded      38s                xgboostjob-operator  XGBoostJob xgboost-dist-iris-test-predict is successfully completed.
 ```
+
+**Start the distributed XGBoost train to store the model locally**
+Before proceeding with training we will create a PVC to store the model trained.
+Creating pvc : 
+create a yaml file with the below content 
+pvc.yaml
+```
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: xgboostlocal
+spec:
+  storageClassName: glusterfs
```

Comment:
Why glusterfs? It could be any storageClass. Please mention that glusterfs is just one of the choices